x-pack/plugins/infra/public/containers/libs/nodes_to_wafflemap.ts

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
I wonder if we could find a better place for these files, because this is actually the directory that contains the "libs" container (which injects a reference to the libs into the React context).

Since these function are used by the map container, they could live in its directory, for example.

Reply:
Ha Ha... that's where it lived originally. I will move it back.

Reply:
thanks, let me know when I should take another look